modules/dr-restoring-etcd-quorum-ha.adoc

```diff
@@ -21,26 +26,132 @@ You might experience data loss if the host that runs the restoration does not ha
 
 . Select a control plane host to use as the recovery host. You run the restore operation on this host.
 
-. Using SSH,  connect to the chosen recovery node and run the following command to restore etcd quorum:
+.. List the running etcd pods by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-etcd -l app=etcd --field-selector="status.phase==Running"
+----
+
+.. Choose a pod and run the following command to obtain its IP address:
++
+[source,terminal]
+----
+$ oc exec -n openshift-etcd <etcd-pod> -c etcdctl -- etcdctl endpoint status -w table
+----
++
+Note the IP address of a member that is not a learner and has the highest Raft index.
+
+.. Run the following command and note the node name that corresponds to the IP address of the chosen etcd member:
++
+[source,terminal]
+----
+$ oc get nodes -o jsonpath='{range .items[*]}[{.metadata.name},{.status.addresses[?(@.type=="InternalIP")].address}]{end}'
+----
+
+. Using SSH, connect to the chosen recovery node and run the following command to restore etcd quorum:
 +
 [source,terminal]
 ----
 $ sudo -E /usr/local/bin/quorum-restore.sh
 ----
++
+After a few minutes, the nodes that went down are automatically synchronized with the node that the recovery script was run on. Any remaining online nodes automatically rejoin the new etcd cluster created by the `quorum-restore.sh` script. This process takes a few minutes.
 
 . Exit the SSH session.
 
+. Return to a three-node configuration if any nodes are offline. Repeat the following steps for each node that is offline to delete and re-create them. After the machines are re-created, a new revision is forced and etcd automatically scales up.
++
+** If you use a user-provisioned bare-metal installation, you can re-create a control plane machine by using the same method that you used to originally create it. For more information, see "Installing a user-provisioned cluster on bare metal".
++
+[WARNING]
+====
+Do not delete and re-create the machine for the recovery host.
+====
++
+** If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps:
++
+[WARNING]
+====
+Do not delete and re-create the machine for the recovery host.
+
+For bare-metal installations on installer-provisioned infrastructure, control plane machines are not re-created. For more information, see "Replacing a bare-metal control plane node".
+====
+
+.. Obtain the machine for one of the offline nodes.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api -o wide
+----
++
+Example output:
```

Comment:
```suggestion
Example output:
```
I can't find the formal reference but I "think"  the  'Example output' should have a leading period? For example, `.Example output`. Feel free to ignore this though because I can't find the directive; it is just in my mind. :-)

Reply:
I think you're right. This came from the previous docs and I didn't notice. Thanks for catching that!